packages/server/src/question/question.subscriber.ts

```diff
@@ -62,6 +63,29 @@ export class QuestionSubscriber
   }
 
   async afterInsert(event: InsertEvent<QuestionModel>): Promise<void> {
+    const numberOfQuestions = await QuestionModel.openInQueue(
+      event.entity.queueId,
+    )
+      .where('question.status IN (:...openStatus)', {
+        openStatus: [OpenQuestionStatus.Drafting, OpenQuestionStatus.Queued],
+      })
+      .getCount();
+
+    const staff = (
```

Comment:
Can you grab the stafflist inside the if statement so we don't waste a db call when there's no need to notify